clouds/snowflake/modules/test/h3/H3_CENTER.test.js

```diff
@@ -0,0 +1,38 @@
+const { runQuery } = require('../../../common/test-utils');
+
+test('Returns NULL with invalid parameters', async () => {
+    const query = `
+        WITH ids AS
+        (
+            SELECT 1 AS id, NULL as hid UNION ALL
+            SELECT 2 AS id, 'ff283473fffffff' as hid
+        )
+        SELECT
+            id,
+            H3_CENTER(hid) as bounds
+        FROM ids
+        ORDER BY id ASC
+    `;
+
+    const rows = await runQuery(query);
+    expect(rows.length).toEqual(2);
+    expect(rows[0].BOUNDS).toEqual(null);
+    expect(rows[1].BOUNDS).toEqual(null);
+});
+
+test('Returns NULL the expected geography', async () => {
+    const query = `
+        WITH ids AS
+        (
+            SELECT 1 AS id, H3_CENTER('85283473fffffff') as bounds, TO_GEOGRAPHY('POINT(-121.9763759725512 37.34579337536848)') AS expected UNION ALL
```

Comment:
@@SF_SCHEMA@@.H3_CENTER

Reply:
I would say this is not necessary. I introduced the schema in the case of buffer because apparently they have introduced their own version and there was a conflict.
https://docs.snowflake.com/en/sql-reference/functions/st_buffer

Reply:
Good news. Native ST_BUFFER. Agree with your comment